dev/ci/ci-coqtail.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+ci_dir="$(dirname "$0")"
+. "${ci_dir}/ci-common.sh"
+
+FORCE_GIT=1
```

Comment:
Why force git?

Reply:
It was just part of the file I initially copied.